lecture/forms.py

```
@@ -20,3 +29,24 @@ def save(self, commit=True, lecture=None):
         if commit:
             instance.save()
         return instance
+
+class LectureVideoForm(forms.ModelForm):
+    class Meta:
+        model = LectureVideo
+        fields = ['video_file', 'lecture']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)